src/ripple/app/ledger/impl/InboundLedger.cpp

```diff
@@ -66,20 +66,25 @@ enum
 // millisecond for each ledger timeout
 auto constexpr ledgerAcquireTimeout = 2500ms;
 
-InboundLedger::InboundLedger(Application& app, uint256 const& hash,
-    std::uint32_t seq, Reason reason, clock_type& clock)
-    : PeerSet (app, hash, ledgerAcquireTimeout, clock,
-        app.journal("InboundLedger"))
-    , mHaveHeader (false)
-    , mHaveState (false)
-    , mHaveTransactions (false)
-    , mSignaled (false)
-    , mByHash (true)
-    , mSeq (seq)
-    , mReason (reason)
-    , mReceiveDispatched (false)
+InboundLedger::InboundLedger(
+    Application& app,
+    uint256 const& hash,
+    std::uint32_t seq,
+    Reason reason,
+    clock_type& clock)
+    : PeerSet(app, hash, ledgerAcquireTimeout, app.journal("InboundLedger"))
```

Comment:
Is it possible to tweak `clang-format` here so that _either_ the argument list or the initializer list gets an extra indent?

Reply:
We probably want to get that change into the format PR before it gets merged. I'll note the argument list and initializer list are already distinguishable by the leading separators for each initializer (either `:` or `,`).

Reply:
We can set `ConstructorInitializerIndentWidth` so that the initializers _always_ get an extra indent (even if there no parameters).